### PR TITLE
3 setup rtd

### PR DESCRIPTION
**Pull request by emlynjdavies**
RTD is now setup here: https://pyopia.readthedocs.io

Docs will build automatically on the main branch each time that is updated. Doc strings are defined under function definitions and will then be detected automatically and become part of this main documentation.

Currently RDT is setup only to build for this branch until this PR is approved.

### CODE REVIEW

**Comment by emlynjdavies** [comment]
There are a lot of unnecessary commits here, as I was trying to get auto-building of RDT to work. So I suggest a squash merge once this PR is approved.

@animmosmith Please could you have a check if you are happy with the basic into paragraph on RDT, here: https://pyopia.readthedocs.io (and here in this repo: https://github.com/SINTEF/pyopia/blob/3-setup-rtd/docs/source/index.rst). We can improve this description as this develops, but just want to check that this could be an OK start?

**Comment by animmosmith** [comment]
Thanks @emlynjdavies - looks good, but would suggest removing the holography hyperlink as other reconstruction methods may also be used!

**Comment by emlynjdavies** [comment]
> Thanks @emlynjdavies - looks good, but would suggest removing the holography hyperlink as other reconstruction methods may also be used!

@animmosmith OK, thanks. We should be good now. Could you try to approve the review request and see if it lets you merge the PR? (I suggest a squashed merge)

**Review by animmosmith — APPROVED** [review]

**Pull request merged**
